Makefile

```diff
@@ -1,5 +1,5 @@
 GOFLAGS :=
-DOCKER_ORG ?= $(USER)
+IMAGE_REPOSITORY_NAME ?= openshift
```

Comment:
as we have started the bikeshedding I don't think this is the correct nomenclature

repository is openshift/release, this is an org